tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -749,28 +749,18 @@ def args_dict_to_list(cli_parser, **kwargs):
     return result
 
 
-def pack_params_to_args_namespace(**kwargs):
-    fe_manager = FrontEndManager()
-    cli_parser = get_all_cli_parser(fe_manager)
-    argv = cli_parser.parse_args(args_dict_to_list(cli_parser, **kwargs))
-
-    all_params = {}
-    for key, value in mo_convert_params.items():
-        all_params.update(value)
-
-    for key, value in kwargs.items():
-        if key not in argv and key not in all_params.keys():
-            raise Error("Unrecognized argument: {}".format(key))
-        if value is not None:
-            setattr(argv, key, value)
-    send_params_info(argv, cli_parser)
-
+def get_non_default_params(argv, cli_parser):
+    # make dictionary with parameters which have non-default values to be serialized in IR in rt_info
     non_default_params = {}
-    for arg in vars(argv):
+    for arg in vars(argv).keys():
         arg_value = getattr(argv, arg)
         if arg_value != cli_parser.get_default(arg):
-            non_default_params[arg] = depersonalize(arg_value, arg)
-    return argv, non_default_params
+            value = depersonalize(arg_value, arg)
+            # Skip complex classes in params to prevent
+            # serializing it to rt_info
+            if isinstance(value, (str, bool)):
```

Comment:
should we return other standard types?

Reply:
Most of the parameters are converted to string before this line, but for 'batch' and 'scale' conversion to string is not happening, they keep numeric values. So I added numbers.Number to this check and added 'batch' and 'scale' to meta data test.